ee/allowedcmd/cmd.go

```diff
@@ -110,3 +110,20 @@ func IsNixOS() bool {
 	checkedIsNixOS = true
 	return isNixOS
 }
+
+func Launcher(ctx context.Context, arg ...string) (*TracedCmd, error) {
```

Comment:
I think there are a couple places (the interactive checkup, the desktop runner, the windows update table) that could use this new function, maybe worth filing an issue to update 'em?

Reply:
sounds like a good idea to me, will do!